cattle/__init__.py

```diff
@@ -212,6 +212,10 @@ def agent_instance_cattle_home():
         return default_value('AGENT_INSTANCE_CATTLE_HOME', '/var/lib/cattle')
 
     @staticmethod
+    def container_state_dir():
+        return path.join(Config.state_dir(), 'containers')
```

Comment:
How important is this directory in being persisted?  It is possible this directory could be deleted between agent restarts